setup.py

```diff
@@ -0,0 +1,105 @@
+# This file is part of REANA.
+# Copyright (C) 2018, 2018 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation; either version 2 of the License, or (at your option) any later
+# version.
+#
+# REANA is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# REANA; if not, write to the Free Software Foundation, Inc., 59 Temple Place,
+# Suite 330, Boston, MA 02111-1307, USA.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization or
+# submit itself to any jurisdiction.
+
+"""REANA-Workflow-Engine-Serial."""
+
+from __future__ import absolute_import, print_function
+
+import os
+import re
+
+from setuptools import find_packages, setup
+
+readme = open('README.rst').read()
+history = open('CHANGES.rst').read()
+
+tests_require = [
+    'check-manifest>=0.25',
+    'coverage>=4.0',
+    'httpretty>=0.8',
+    'isort>=4.2.2',
+    'pydocstyle>=1.0.0',
+    'pytest-cache>=1.0',
+    'pytest-cov>=1.8.0',
+    'pytest-pep8>=1.0.6',
+    'pytest>=2.8.0,<3.0.0'
+]
+
+extras_require = {
+    'docs': [
+        'Sphinx>=1.4.4',
+        'sphinx-rtd-theme>=0.1.9',
+    ],
+    'tests': tests_require,
+}
+
+extras_require['all'] = []
+for key, reqs in extras_require.items():
+    if ':' == key[0]:
+        continue
+    extras_require['all'].extend(reqs)
+
+setup_requires = [
+    'pytest-runner>=2.7',
+]
+
+install_requires = []
+
+packages = find_packages()
+
+
+# Get the version string. Cannot be done with import!
+with open(os.path.join('reana_workflow_engine_serial',
+                       'version.py'), 'rt') as f:
+    version = re.search(
+        '__version__\s*=\s*"(?P<version>.*)"\n',
+        f.read()
+    ).group('version')
+
+setup(
+    name='reana-workflow-engine-serial',
+    version=version,
+    description=__doc__,
+    long_description=readme + '\n\n' + history,
+    author='REANA',
+    author_email='info@reana.io',
+    url='https://github.com/reanahub/reana-workflow-engine-serial',
+    packages=['reana_workflow_engine_serial', ],
+    zip_safe=False,
+    install_requires=install_requires,
+    extras_require=extras_require,
+    setup_requires=setup_requires,
+    tests_require=tests_require,
+    classifiers=[
+        'Development Status :: 3 - Alpha',
+        'Environment :: Web Environment',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: GNU General Public License v2 (GPLv2)',
+        'Operating System :: OS Independent',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
```

Comment:
We should aim at 3.using Python 3.6... 2.7 is only there "historically" for Yadage etc.